llvm/include/llvm/Cheerp/Writer.h

```diff
@@ -692,6 +692,10 @@ class CheerpWriter final : public CheerpBaseWriter
 	 * Compile the function for growing the wasm linear memory
 	 */
 	void compileGrowMem();
+	/**
+	 * Compile the atomic functions
+	 */
+	void compileAtomicFunctions();
```

Comment:
You probably wanted to add this in a later commit, alongside its definition

Reply:
Correct. Moved it to the right commit.